Documentation/D2.md

```diff
@@ -21,17 +21,30 @@ During our first meeting we discovered that:
+    * Repository size and commit size could be a potential factor. When pull requests are accepted they are merged which in large repositories and commits could take longer to complete. Because we are not interested in the performance of githubs algorythms we will be limiting the number of lines needed to merge to no more than 20 line changes.
 * Cost and marketability (A survey with standard questions that contribute to the goal of obtaining the level of interest, user acceptance, and device pricing)
+  * Experiment: We will attempt to determine the end user marketability of our device by surveying a population of developers familure with github.
+  * Procedure: Using an anonymous written(digitial or paper) survey with questions pertaining to user acceptance of the device using a veriety of dichotomous and Likert scales. The Likert scales will be built using current standards according to reasearch using a 5 point scale with the polor lables being delight and frustration.
+  * Expected Results: We expect the results to revieal that users will be more motivated to preform valid actions, and find programming more enjoyable or whimsicle.
   * Independent Variables:
     * The survey will not be peer reviewed and thus subject to a degree of bias. We will however take our best efforts not to ask leading questions and ask the questions in an unbiased order.
     * The survey sample size will likely be small and could contain sampling bias. While we will attempt to survey a variety of professionals, our access to developers within Colorado State University's student body is much more readily available.
 * Power consumption (Using standard RPi power statistics to estimate power consumption)
+  * Experiment: We will measure the power consumption of the device during a standard american work period of 9 hours (8 hours of working time + an hour lunch break).
+  * Procedure:
+  * Expected Results:
   * Independent Variables:
     * Measuring power consumption relies on existing tools that have a degree of error. We will include the error tolerances in our findings.
 * Security evaluation (Evaluate any security concerns using published documents for existing methods and hardware. Running a network sniffing tool (wire shark) to determine what, if any information is leaked through network traffic)
+  * Experiment: We will evaluate the information the device leaks, and broadcasts during typical use.
+  * Procedure: Using wireshark packet analysis to monitor web communications. We will review these packets for any sensitive information leaked
+  * Expected Results: We expect no sensitive information to be leaked. Github is mainly an open source platform, code and repository names are largly public, it is also a trusted platform and we expect that we will not find that the service leaks any sensitive information. Sensitive information would be webhook private security keys and passwords.
```

Comment:
Change "Sensitive information would be webhook private security keys and passwords." to "However, we may find that information transmitted over Wifi between the client and the device may contain sensitive data such as source code or account information such as private security keys and passwords.